.github/workflows/create-release-candidate-pr.yml

```diff
@@ -0,0 +1,100 @@
+###########################################################################################################
+#
+# This job creates a release candidate branch based on master, and opens a PR from it.
+#
+# The branch will have the version bumped in all necessary parts of the code,
+# and it will generate the NEWS.md content from the NEWS snippets.
+#
+# Organization members have the chance to add or modify the commits on the branch, if necessary.
+#
+# The job will also generate tarballs, and reference them in the PR's description.
+#
+# The job can be run multiple times, it will always create a new release candidate branch and PR each time.
+# E.g.: v1.2.3rc0, v1.2.3rc1, ...
+#
```

Comment:
I think there's no point in starting again from "master". I my mind rc (=release candidate) releases happen during the preparation of an upcoming public release and each subsequent rc should contain everything up to that point.

E.g. rc0 (or rc1) happens at the initial fork of the release branch, subsequent rcs happen when a bug is found and fixed during the release specific testing. As these fixes accumulate we might cut more rc releases up to a point where we finish and finish the one with the final version number.

This boils down to the creation of a "release" branch once and cutting rc releases potentially multiple times from this same branch.

Reply:
This seems like a valid idea, just a bit different way of working, that I imagined. Let's talk about this tomorrow on the meeting.
Thanks!

Reply:
We discussed this, and ended up changing the job name to "Version bump", and reusing the git branch every time, the job is re-run. We don't allow to run the job, while there is a "Version bump" PR open.